nucypher/characters/base.py

```diff
@@ -74,6 +76,7 @@ class Character(Learner):
 
     def __init__(self,
                  domains: Set = None,
+                 node_class: object = None,
```

Comment:
Might just be personal preference, but this would be clearer to me if it was `known_node_class`.

Reply:
Hmmm, yeah.

Reply:
:heavy_check_mark: